packages/@aws-cdk/aws-appmesh/test/test.mesh.ts

```diff
@@ -236,7 +236,6 @@ export = {
                     HealthyThreshold: 3,
                     IntervalMillis: 5000,
                     Path: '/',
-                    Port: 8080,
```

Comment:
Again, this change is concerning to me. Why are we changing this?

Reply:
`port` was dropped from the interface as we only allow it to be the same as the listener, which is the default when port is not specified. Seems like a bad customer experience to have an option to configure it